packages/extension-store-locator/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@salesforce/extension-store-locator",
+  "version": "1.0.0-dev",
+  "scripts": {
+    "test": "pwa-kit-dev test"
+  },
+  "peerDependencies": {
+    "@chakra-ui/react": "^2",
+    "@loadable/component": "^5",
+    "@salesforce/commerce-sdk-react": ">=1.0.0",
+    "@salesforce/pwa-kit-react-sdk": ">=3.0.0",
+    "@salesforce/pwa-kit-runtime": ">=3.0.0",
+    "prop-types": "^15",
```

Comment:
Do we need prop types in a typescript project?

Reply:
after i convert that one last file to typescript, i'll remove it, good catch